.gitignore

```diff
@@ -1,4 +1,5 @@
 /target
 .vscode/
 logs/
-.temp/
\ No newline at end of file
+.temp/
+Cargo.lock
```

Comment:
Cargo.lock shouldn't be in `.gitignore`, it should be added there only for libraries.

https://github.com/rust-lang/cargo/issues/315

Reply:
I've updated `Cargo.lock` instead.